graphs/DirectRuntime.fbp

```diff
@@ -7,8 +7,9 @@ HoldConnection(ui/HoldRuntimeConnection) CONTEXT -> IN ConnectedContext(core/Mer
 'false' -> REQUIREGRAPHS HoldConnection
 
 # Read runtime definition from context and connect
-'runtime' -> KEY GetDefinition(objects/GetObjectKey)
+'runtime' -> STRING SendRuntimeKey(strings/SendString) OUT -> KEY GetDefinition(objects/GetObjectKey)
```

Comment:
What is this SendString? Looks just like `core/Kick`...

Reply:
Yes, very similar, but with two differences:

* SendString is typed, so we know what we're sending is a string
* SendString sends on data packet, Kick sends on complete stream

Reply:
Ok, thanks for explaining